redefining O_LARGEFILE like this breaks it on aarch64,
because it is 0400000(!).
musl defines this correctly, so don't redefine it
https://github.com/9fans/plan9port/issues/619
--
diff --git a/src/cmd/9pfuse/main.c b/src/cmd/9pfuse/main.c
index 4fa330a..2b66875 100644
--- a/src/cmd/9pfuse/main.c
+++ b/src/cmd/9pfuse/main.c
@@ -38,7 +38,7 @@
  * On ARM however, the O_LARGEFILE is set correctly.
  */
 
-#if defined(__linux__) && !defined(__arm__)
+#if defined(__GLIBC__)
 #  undef O_LARGEFILE
 #  define O_LARGEFILE 0100000
 #endif
